src/Avalonia.X11/X11FocusProxy.cs

```diff
@@ -0,0 +1,78 @@
+using System;
+using System.Diagnostics;
+using static Avalonia.X11.XLib;
+
+namespace Avalonia.X11
+{
+    /// <summary>
+    ///     An invisible X window that owns the input focus and forwards events to the owner window. 
+    /// </summary>
+    /// <remarks>
+    ///     <para>
+    ///         This is a known Linux technique for an auxiliary invisible window to hold the input focus 
+    ///         for the main window. It is required by XEmbed protocol, but it also works for regular cases 
+    ///         that don't imply embedded windows.
+    ///     </para>
+    /// </remarks>
+    /// 
+    /// <see href="https://specifications.freedesktop.org/xembed-spec/xembed-spec-latest.html" />
+    /// <see href="https://gitlab.gnome.org/GNOME/gtk/-/blob/3.22.30/gdk/x11/gdkwindow-x11.c#L823 />
+    internal class X11FocusProxy
+    {
+        private const int InvisibleBorder = 0;
+        private const int DepthCopyFromParent = 0;
+        private readonly IntPtr VisualCopyFromParent = IntPtr.Zero;
```

Comment:
```suggestion
        private readonly IntPtr _visualCopyFromParent = IntPtr.Zero;
```
I think it's better to use the existing code style.